t/opbasic/arith.t

```diff
@@ -429,16 +429,20 @@ $a = (97656250000000000 % $1);
 $b = (97656250000000000 % "$1");
 print "not "x($a ne $b), "ok ", $T++, qq ' - something % \$1 vs "\$1"\n';
 
+unless (eval { require Config }) {
```

Comment:
Why not just «use Config;»?

Reply:
`t/opbasic/*` is run by minitest and shouldn't depend on Config loading successfully.

It shouldn't be loading test.pl either (which is partly my mistake).

Reply:
> Why not just «use Config;»?

My change uses `eval { require ... }` because `arith.t` already had it in `if ($^O eq 'VMS')` block, and I first started this change with moving this `eval` out of the if-block.
And I didn't changed `require` to `use` because I think `arith.t` is a basic test and should be run (at least partially) even if `Config` couldn't be loaded successfully.

I think it might be better to split Config-dependent tests to a separate file, for example `t/op/arith2.t`.